gallery/poisson3d/ceed-poisson3dapply.h

```diff
@@ -42,6 +42,7 @@ CEED_QFUNCTION(Poisson3DApply)(void *ctx, const CeedInt Q,
     // 0 5 4
     // 5 1 3
     // 4 3 2
+    // *INDENT-OFF*
     const CeedScalar dXdxdXdxT[3][3] = {{qd[i+0*Q],
```

Comment:
Astyle really hates writing this style and wants to explode the array so that braces each have their own line.

Reply:
We could consider packing such arrays via a loop or even a macro/function call (though we'd have to make it available on the CUDA/JIT side as well).

Reply:
Right, I think that would be preferable so we can minimize the use of `INDENT-OFF`

Reply:
Updated